cadquery/cq.py

```diff
@@ -423,7 +422,7 @@ def vals(self) -> List[CQObject]:
         :rtype: list of occ_impl objects
         :returns: the values of the objects on the stack.
 
-        Contrast with :py:meth:`all`, which returns CQ objects for all of the items on the stack
+        Contrast with :meth:`all`, which returns CQ objects for all of the items on the stack
```

Comment:
These types of entries seem to work with `:py:`, so I'm curious what the motivation is to remove it. Was it causing some sort of error?

Reply:
@jmwright You are correct, it's not an error to specify the domain here.  Since the default domain is already Python (py), I thought it would be simpler and less verbose to omit the domain.